src/components/Login/Login.js

```diff
@@ -82,6 +89,13 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
 
   const formMessage = useMemo(() => getForgotErrorMessage(location), [location]);
 
+  const activationInProgress = useMemo(() => isActivactionInProgress(location), [location]);
+
+  if(activationInProgress && typeof(window.gtag) === 'function') {
```

Comment:
Why `window.gtag` could be not a function?

Reply:
only validation if library doesnt load propertly

Reply:
👍 I understand. We do not want to break our application if the library's CDD is down.